flow/shared/alerting/alerting.go

```diff
@@ -55,9 +58,20 @@ func NewAlerter(catalogPool *pgxpool.Pool) (*Alerter, error) {
 	if catalogPool == nil {
 		return nil, errors.New("catalog pool is nil for Alerter")
 	}
-
+	snsTopic := os.Getenv("TELEMETRY_AWS_SNS_TOPIC_ARN")
+	var snsMessageSender telemetry.Sender
+	if snsTopic != "" {
```

Comment:
can we log something like - "successfully registered telemetry client" or something.